Sources/XiEditor/Theme.swift

```diff
@@ -76,23 +76,23 @@ extension Theme {
             )
     }
 
-    init(jsonObject dict: [String: AnyObject]) {
```

Comment:
This is just a big rename from `dict` to `json` and changing `AnyObject` to `Any`.